client/interfaces.go

```diff
@@ -64,9 +64,18 @@ type Interfaces struct {
 	Slots []Slot `json:"slots"`
 }
 
+// InterfaceInfo holds information about a given interface and its instances.
+type InterfaceInfo struct {
+	MetaData InterfaceMetaData `json:"meta-data"`
+	Plugs    []Plug            `json:"plugs,omitempty"`
+	Slots    []Slot            `json:"slots,omitempty"`
```

Comment:
Can we merge InterfaceInfo and InterfaceMetaData into a single type named Interface?

Note that the current Interfaces type holds Plugs and Slots, exactly the same as above.